pkg/plugin/manager.go

```diff
@@ -183,3 +216,122 @@ func (m *manager) GetBlockStore(name string) (cloudprovider.BlockStore, error) {
 
 	return blockStore, nil
 }
+
+// GetBackupActions returns all backup.BackupAction plugins.
+// These plugin instances should ONLY be used for a single backup
+// (mainly because each one outputs to a per-backup log),
+// and should be terminated upon completion of the backup with
+// CloseBackupActions().
+func (m *manager) GetBackupItemActions(backupName string, logger logrus.FieldLogger, level logrus.Level) ([]backup.ItemAction, error) {
+	// have we already discovered all external BackupAction binaries? if not, discover them.
+	var binaries []string
+
+	if err := func() error {
+		m.pluginBinariesLock.Lock()
+		defer m.pluginBinariesLock.Unlock()
+
+		binaries = m.pluginBinaries[PluginKindBackupItemAction]
+		if binaries == nil {
+			binaries = make([]string, 0)
+			if _, err := os.Stat(pluginDir); err == nil {
```

Comment:
Maybe log a warning that the plugin dir doesn't exist so no external plugins will be loaded?